src/features/sandbox-banner/__tests__/SandboxBanner.stories.test.tsx

```diff
@@ -0,0 +1,6 @@
+describe('features/sandbox-banner/SandboxBanner', () => {
+    test.each([['features-sandboxbanner--basic']])('looks visually correct when using story %s', async id => {
```

Comment:
Can you add another visual test for the hover state (using `takeScreenshotAfterInput`)?